kubeflow/rebuild-image.sh

```diff
@@ -4,24 +4,31 @@ export AWS_PROFILE
 
 aws ecr get-login --no-include-email | bash
 
-set -ex
+. ../config
+
+cp ../lib.sh .
+. lib.sh
+
+check_config "IMAGE COMMON_IMAGE genie_version thingtalk_version bootleg_version genienlp_version "
```

Comment:
Kubeflow runs with `imagePullPolicy: IfNotPresent`, which means every docker image _must_ have a different tag, or it might load the wrong one. Using the config file is inconvenient and risks pushing to the wrong tag.